src/main/java/org/scijava/plugins/scripting/python/PythonScriptEngine.java

```diff
@@ -78,10 +82,13 @@ public Object eval(final String script) throws ScriptException {
 				.filter(obj -> "PythonScriptRunner".equals(objectService.getName(obj)))//
 				.findFirst();
 		if (!pythonScriptRunner.isPresent()) {
+			// Try to help user by running OptionsPython plugin
+			if (commandService != null) {
+				commandService.run(OptionsPython.class, true);
```

Comment:
I have mixed feelings about this. It introduces an assumption that this script engine will always be used in a GUI environment, which it won't necessarily be. It feels like a violation of separation of concerns to me. Similarly, the error message below is much friendlier but less correct. I'm OK leaving these changes in place for now, but they may need further adjustment or removal if other scenarios arise where they make sense to do.